lib/hammer_cli_foreman/hostgroup.rb

```diff
@@ -15,11 +15,9 @@ def self.included(base)
       base.option "--puppet-ca-proxy", "PUPPET_CA_PROXY_NAME", _("Name of puppet CA proxy")
       base.option "--puppet-proxy", "PUPPET_PROXY_NAME",  _("Name of puppet proxy")
       base.option "--parent", "PARENT_NAME",  _("Name of parent hostgroup")
-      base.option ["--root-password", "--root-pass"], "ROOT_PASSWORD",  _("Root password"),
-                  deprecated: { '--root-pass' => _("Use --root-password instead") }
+      base.option ["--root-password"], "ROOT_PASSWORD",  _("Root password")
       base.option ["--ask-root-password", "--ask-root-pass"], "ASK_ROOT_PW", "",
```

Comment:
@adiabramovitch, if you're removing dep warning here, you then probably want to remove the `--ask-root-pass` option as well :) You can use Refs #22574 commit message for the following PR.